tools/build_utils.py

```diff
@@ -220,16 +220,14 @@ def build_tensorflow(tf_version, venv_dir, src_dir, artifacts_dir, target_arch,
         "--config=opt",
         "--config=noaws",
         "--config=nohdfs",
-        "--config=noignite",
-        "--config=nokafka",
         "--config=nonccl",
     ]
     # Build the python package
-    if (tf_version == "v2.0.0"):
+    if (tf_version.startswith("v2.")):
```

Comment:
With these changes the user will be able to build TF2.x using build_tf script and use it with --use_tf_from_location build option, correct?

Reply:
Yes.